src/components/App/App.tsx

```diff
@@ -104,13 +99,9 @@ const App = () => {
 }
 
 export default App;
-// DONE:
```

Comment:
Thanks for taking these out! Per Kayla's earlier comments - it would be a good habit to get into to take out all commented code before merging to main.